convene-web/app/models/utility_connection.rb

```diff
@@ -0,0 +1,24 @@
+# Links a {Utilities::Utility} to a {Space} or {Person}.
+class UtilityConnection < ApplicationRecord
+  # Which Space the Furniture is connected to.
+  belongs_to :space
+
+  # Human-friendly name for disambiguating when a single kind of Utility has
+  # multiple Connections.
+  attribute :name
+  attribute :description
+
+  # Which type of {Utility} is connected, for example Airtable or  Asana or
+  # Jitsi-Meet or BBB or Plaid or Stripe.
+  attribute :utility_type
```

Comment:
Noodling out loud: would this eventually be an enum with a registry?

Reply:
Yea, I wonder if there's a Registry class provided by Rails  that we could use for this...

Reply:
There is not a registry class provide by Rails for this :(